authors/apps/authentication/serializers.py

```diff
@@ -11,10 +11,7 @@ class RegistrationSerializer(serializers.ModelSerializer):
     # Ensure passwords are at least 8 characters long, no longer than 128
```

Comment:
These changes don't affect this file too.

Reply:
I am not getting @esirK, I have not done any changes to the serializers file. It was yapf package that formatted and brought all these changes. The feature is working, generating token successfully. 🔧 .

Reply:
I get what happened. I would still ask you to review the files you `git add`. Only add files related to the task at hand.

Reply:
Kindly remove it since it doesn't correspond to this task.